posthog/plugins/access.py

```diff
@@ -3,28 +3,52 @@
 
 from django.conf import settings
 
-from posthog.models import organization
 from posthog.models.organization import Organization
 
 
-# We disable Plugins on Cloud, except for whitelisted organizations
-# Disregarding this in TEST mode, so that we can be sure plugins actually work in EE if/when needed
-def guard_cloud(organization_or_id: Optional[Union[Organization, UUID, str]]):
-    organization_id: Optional[str] = (
-        None
-        if not organization_or_id
-        else str(organization_or_id if isinstance(organization_or_id, (str, UUID)) else organization_or_id.id)
+def can_root_plugins_via_api(
```

Comment:
This file does not seem to have any unit tests

Reply:
Added